extensions/vertx-web/deployment/src/test/java/io/quarkus/vertx/web/mutiny/SyncRouteTest.java

```diff
@@ -87,6 +118,13 @@ Person getPersonUtf8(RoutingContext context) {
             return new Person("neo", 12345);
         }
 
+        @Route(path = "manual-content-type", produces = "application/json")
```

Comment:
`manual-content-type` is not a good path for this route. Maybe `default-content-type`?